persistent-qq/persistent-qq.cabal

```
@@ -1,6 +1,6 @@
 cabal-version: 1.12
 name:           persistent-qq
-version:        2.12.0.6
+version:        2.12.1
```

```suggestion
version:        2.12.0.7
```

Since API hasn't changed, we can do a patch bump